From 7c8d79306bce85e70cb9fb21a045ba881174a5d2 Mon Sep 17 00:00:00 2001
From: Huibin Hong <huibin.hong@rock-chips.com>
Date: Mon, 13 Jun 2016 09:14:34 +0800
Subject: [PATCH] arm64: dts: rockchip: add burst mode for rk3368

Because rk3368 pl330 dma controller doesn't support single
mode, so it is necessary to set peripherals-req-type-burst.

Please refer to:
Commit 8e770f371cc2
"dmaengine: pl330: add burst mode according to dts config"

Change-Id: I44de28cca0085bc3d8f25a5913dbb527c36d8f83
Signed-off-by: Huibin Hong <huibin.hong@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index f3ad29070dc5..058fd911288a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -285,6 +285,7 @@
 			clocks = <&cru ACLK_DMAC_PERI>;
 			clock-names = "apb_pclk";
 			arm,pl330-broken-no-flushp;
+			peripherals-req-type-burst;
 		};
 
 		dmac_bus: dma-controller@ff600000 {
@@ -296,6 +297,7 @@
 			clocks = <&cru ACLK_DMAC_BUS>;
 			clock-names = "apb_pclk";
 			arm,pl330-broken-no-flushp;
+			peripherals-req-type-burst;
 		};
 	};
 
-- 
2.35.3

